kmod/scoutfs-kmod.spec.in

```diff
@@ -104,3 +104,14 @@ depmod -a
 %clean
 rm -rf %{buildroot}
 
+%preun
+# stash our modules for postun cleanup
+SCOUTFS_RPM_NAME=$(rpm -q %{name} | grep "%{version}-%{release}")
+rpm -ql $SCOUTFS_RPM_NAME | grep '\.ko$' > /var/run/%{name}-modules || true
+
+%postun
+/sbin/ldconfig
+
+if [ -x /sbin/weak-modules ]; then
+    cat /var/run/%{name}-modules | /sbin/weak-modules --remove-modules --no-initramfs
```

Comment:
probably also should `rm /var/run/%{name}-modules` here.

Reply:
Good catch, agreed!